.circleci/config.yml

```diff
@@ -84,9 +84,6 @@ jobs:
           # Given the RAM limits on CI (8GB), max-workers=7 cause OOMs.
           # Having 5 or 6 workers doesn't seem to decrease compilation time significantly, as compared to just 4.
           command: ./gradlew --max-workers=4 compileJava compileKotlin
-      - run:
-          name: Run post-compile checks
```

Comment:
is there any significant difference on the pipeline duration?

Reply:
Nopes, it's just a matter of getting stuff more unified